app/src/main/res/values/strings.xml

```diff
@@ -105,6 +105,7 @@
   <string name="preferences">Preferences</string>
   <string name="profile_progress_activity_title">Profile Progress Page</string>
   <string name="search">Search</string>
+  <string name="force_crash_occurred">Force crash occurred</string>
```

Comment:
I suggest using a string literal, instead, since these are strings that are meant to be visually displayed to the user (& thus will be translated).

Reply:
Changed string resource to string literal.